src/tiled/mapobjectmodel.cpp

```diff
@@ -445,6 +445,12 @@ void MapObjectModel::emitObjectsChanged(const QList<MapObject *> &objects, const
     }
 }
 
+void MapObjectModel::emitObjectsChanged(const QList<MapObject *> &objects, const MapObjectModel::Column column)
```

Comment:
You don't need to prefix with `MapObjectModel::` there since you're already in the `MapObjectModel` context.